Airports/EGWU/Ground Map/Geo.txt

```diff
@@ -1,1146 +0,0 @@
-EGWU Northolt S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
Please restore this file for archival purposes. It won't be included in the sector file, but we have generally left Ground Map folders intact.

Reply:
okay no worries I have restored this now